src/pipeline.py

```diff
@@ -33,7 +34,7 @@
     },
 )
 
-images = laion_retrieval.apply(
+images = image_urls.apply(
```

Comment:
Should we add a write step here too?

Reply:
We have the write to huggingface hub step in this example.